.circleci/config.yml

```diff
@@ -65,6 +66,7 @@ jobs:
             - ./packages/core-tester-cli/node_modules
             - ./packages/core-transaction-pool/node_modules
             - ./packages/core-transaction-pool-mem/node_modules
+            - ./packages/core-transaction-pool-redis/node_modules
```

Comment:
Don't think we have redis anymore :thinking:

Reply:
Already reverted those :danstop: it got auto-added because i still had the folder in my local, which i've now removed